server/webhookHandlers/handleTransferWebhook.js

```diff
@@ -50,75 +50,75 @@ const transfersHandler = async (requestBody, io) => {
         const appStatus = await retrieveAppStatus();
         let afterId = appStatus[0].number_of_events;
         let hasEventsToFetch = true;
-        let isFirstCall = true;
         const batchSize = 25; // 25 is the maximum number of events returned
+        let allNewPlaidEvents = [];
         while (hasEventsToFetch) {
           const sycnRequest = {
             after_id: afterId,
             count: batchSize,
           };
           const syncResponse = await plaid.transferEventSync(sycnRequest);
-          const allNewPlaidEvents = syncResponse.data.transfer_events;
-          if (allNewPlaidEvents.length === 0 && isFirstCall) {
-            console.log('no new events');
-            await serverLogAndEmitSocket(webhookCode, 'no events');
-            break;
+          allNewPlaidEvents = await allNewPlaidEvents.concat(
```

Comment:
I don't think that await should be there

Reply:
ooh, thanks.